tests/FakeItEasy.Specs/XBehaveExtensions.cs

```diff
@@ -0,0 +1,70 @@
+namespace FakeItEasy.Specs
+{
+    using System;
+    using System.Diagnostics.CodeAnalysis;
+    using Creation;
+    using Xbehave;
+    using Xbehave.Sdk;
+
+    public static class XBehaveExtensions

```

Comment:
Normally I would agree with @adamralph, but in this case I think we can make an exception to the rule, because this method only makes sense in the context of xBehave